dockers/docker-dhcp-relay/isc-dhcp-relay

```diff
@@ -0,0 +1,6 @@
+SERVERS=""
+
+INTERFACES=""
+
+#-a provides option 82 circuit id information
```

Comment:
I think Manny is explaining that the -a option enables the relay agent to use DHCP's option82 function.
Manny, I think you need to actually specify OPTIONS="-a", right?